WordPress/src/main/res/values/strings.xml

```diff
@@ -140,6 +140,7 @@
     <string name="wp_media_title">WordPress media</string>
     <string name="pick_photo">Select photo</string>
     <string name="pick_video">Select video</string>
+    <string name="pick_media">Select media</string>
```

Comment:
We are using "Add image or video" in Gutenberg, we should use the same terms in both places.

Reply:
Interestingly, when I tested on an Pixel 3a with Android 10, the chooser title seemed to be ignored.